4/Xemsa_4_8.py

```diff
@@ -0,0 +1,27 @@
+#4.8
+"""
+Write a program that reads two numbers aa and bb 
+from the keyboard, calculates and displays the arithmetic 
+mean of all numbers from the segment [a; b], which are 
+multiples of 3.
+In the example below, the arithmetic mean is 
+calculated for numbers on the segment [-5; 12]. The total 
+number of numbers divisible by 3 on this segment is 6: -
+3, 0, 3, 6, 9, 12. Their arithmetic mean is 4.5
+The program receives input to the intervals, within 
+which there is always at least one number that is divisible 
+by 3.
+"""
```

Comment:
This style prefere! Try to use it in the next home works

######################
#Author:
#Date:
#Task:
######################